Submission form created - and  made sure it renders depending on situation

gosuz opened the pull request:
For reference:
- created submission form
- made sure it displays properly depending on the condition (created conditions for it)
- Edited submisssion index.html.erb, pages controller

Note: There's two partials (one for the submission form, and one for the conditional for when to show the form)

gosuz committed:
added submissions controller, submission model, and submission views, and submission form

gosuz committed:
Merge branch 'master' into submission-form-created

gosuz committed:
created submission form, and made sure it displays properly depending on the condition. Edited submisssion index.html.erb, pages controller, submissions controller, user model

HellenTamara commented on app/controllers/submissions_controller.rb:
```diff
@@ -1,5 +1,50 @@
+    submission.user = current_user
+    if submission.save
+      # where do I want to redirect to? home path??
+      redirect_to home_path()
```
Not sure, but maybe the home_path doesn't need the ()

AKW9 replied:
I think it doesn't matter. It works either way

gosuz committed:
Update submissions_controller.rb

Removed some things we didn't need, and added the notice pop up just in case the submission doesn't go through

HellenTamara committed:
Merge branch 'master' into submission-form-created
